patches/net/minecraft/world/entity/EntityType.java.patch

```diff
@@ -130,15 +175,16 @@
          public EntityType<T> build(String p_20713_) {
              if (this.serialize) {
                  Util.fetchChoiceType(References.ENTITY_TREE, p_20713_);
-@@ -1331,7 +_,10 @@
+@@ -1331,7 +_,11 @@
                  this.spawnDimensionsScale,
                  this.clientTrackingRange,
                  this.updateInterval,
 -                this.requiredFeatures
 +                this.requiredFeatures,
 +                velocityUpdateSupplier,
 +                trackingRangeSupplier,
-+                updateIntervalSupplier
++                updateIntervalSupplier,
++                this.requiredFlags.toArray(net.neoforged.neoforge.flag.Flag[]::new)
```

Comment:
If we change this to accepting a set, we won't need the set->array->set conversion.